src/modules/notifications/services/notifications.service.spec.ts

```diff
@@ -4,12 +4,12 @@ import { addMonths } from 'date-fns';
 import { NotificationsService } from './notifications.service';
 
 import {
+	Lookup_Schema_Audience_Type_Enum as AudienceType,
```

Comment:
zouden we deze niet herexporteren in een aparte file?
src/generated/database-aliases.ts

```
export {
  Lookup_Schema_Audience_Type_Enum as AudienceType,
} from '~generated/graphql-db-types-hetarchief';
```

dan moet de importer er niet steeds aan denken dat de naam identiek is, maar kan hij gewoon AudienceType gebruiken en zijn IDE de import laten doen.